src/data.rs

```diff
@@ -1,5 +1,7 @@
 //! Support module for `#[datatest::data(..)]`
-use rustc_test::{Bencher, TDynBenchFn};
+use rustc_test::Bencher;
+#[cfg(feature = "rustc_test_TDynBenchFn")]
+use rustc_test::TDynBenchFn;
```

Comment:
This change looks good, but looks like it is failing CI. :/

Reply:
I had the wrong version, now it's green once again